doc/howtos/building-images.rst

```diff
@@ -207,6 +207,13 @@ Example::
 
 will create both the raw and the VirtualBox appliance images, both compressed.
 
+To get the best benefit out of :command:`bmaptool` when creating your bootable media (see :ref:`booting-and-installation`),
```

Comment:
How about:  The :command:`bmaptool` works best creating your bootable media (see :ref:`booting-and-instalation`) when the corresponding block map ``.bmap`` file is also generated for the image.  This ``.bmap`` file contains empty block and checksum information about the image that lets  :command:`bmaptool` optimize copying the image to your bootable media and verify what was copied.  To do this ...